enterprise/frontend/src/embedding-sdk/components/public/MetabaseProvider.tsx

```diff
@@ -47,7 +52,8 @@ export const MetabaseProviderInternal = ({
   store,
   className,
 }: InternalMetabaseProviderProps): JSX.Element => {
-  const { fontFamily = DEFAULT_FONT } = theme ?? {};
+  const { fontFamily } = theme ?? {};
```

Comment:
I'm removing the default here, it's because otherwise we'll be saving the default font redux, and we'll never fallback to the one coming from the instance settings